
{'date': 'Wed Sep 18 2024 13:10:29.806 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    Historia1()
  module <module> line 8
    	primeiraCnea.vai()
NameError: name 'primeiraCnea' is not defined
'''},
{'date': 'Wed Sep 18 2024 14:16:14.947 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  def HistoriaPrimeiraParte
                            ^
SyntaxError: invalid syntax
'''},